dashbuilder/dashbuilder-backend/dashbuilder-services/src/main/java/org/dashbuilder/transfer/rest/DataTransferResource.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dashbuilder.transfer.rest;
+
+import java.io.IOException;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import org.dashbuilder.transfer.DataTransferExportModel;
+import org.dashbuilder.transfer.DataTransferServices;
+import org.uberfire.io.IOService;
+import org.uberfire.java.nio.file.Paths;
+
+@ApplicationScoped
+@Path("dashbuilder")
+public class DataTransferResource {
+
+    @Inject
+    DataTransferServices dataTransferServices;
+
+    @Inject
+    @Named("ioStrategy")
+    IOService ioService;
+
+    @GET
+    @Path("export")
+    @Produces("application/zip")
+    public Response export() throws IOException {
```

Comment:
Should we really propagate the `IOException`?
In case of an exception, maybe it's worth catching it and returning a more friendly message inside a response error 500.

Reply:
Thank you, Guilherme, I improved the error response when an error happens.